project/plugins.sbt

```diff
@@ -5,6 +5,6 @@ resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases
 addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"         % "2.13.0")
```

Comment:
auto build can be 3.0.0 & you can remove sbt-git-versioning as a dependency after that